src/main/java/net/minecraftforge/common/extensions/IForgeItem.java

```diff
@@ -197,6 +197,8 @@ default boolean onBlockStartBreak(ItemStack itemstack, BlockPos pos, Player play
      * @param count  The amount of time in tick the item has been used for
      *               continuously
      */
+    /** @deprecated use {@link Item#onUseTick(Level,LivingEntity,ItemStack,int)} (note that {@link Item#onUseTick(Level,LivingEntity,ItemStack,int)} can be called with a <pre>useItemRemaining<pre> of <pre>-1<pre>) */
```

Comment:
To reiterate my comment at https://github.com/MinecraftForge/MinecraftForge/pull/9350#discussion_r1132393229,
> This new javadoc should be merged (appended) into the existing javadoc.

Additionally, use `{@code}` instead of `<pre>` tags.